spring-data-cassandra/src/main/java/org/springframework/data/cassandra/core/AsyncCassandraTemplate.java

```diff
@@ -190,11 +191,12 @@ public AsyncCassandraTemplate(AsyncCqlTemplate asyncCqlTemplate, CassandraConver
 		this.entityOperations = new EntityOperations(converter);
 		this.exceptionTranslator = asyncCqlTemplate.getExceptionTranslator();
 		this.statementFactory = new StatementFactory(converter);
+		this.eventDelegate = new EntityLifecycleEventDelegate();
```

Comment:
Nitpicking: I'd prefer to initialize this on declaration since it does not depend on anything in the constructor.